### PR TITLE
Add README.md for the organization

### PR DESCRIPTION
**Pull request by rshym**
The organization's README provides an information gateway with a brief explanation of the main repositories.

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# Welcome to Xen-Troops!
+
+We develop embedded automotive Xen for different hardware platforms and actively contribute to Xen, Linux, OPTEE, Zephyr, and other communities.
```
Comment (lorc) [inline profile/README.md]: "other open source projects". I can't say that we contribute to communities themselves :)

File: profile/README.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+---
+## Where to start
+
+### Build system
+Due to the complex nature of our products and the need to build different OS, we use [moulin](https://github.com/xen-troops/moulin) build system which allows us to build systems containing different OS (Linux, Zephyr, Android) with inter-domain dependencies.
```
Comment (lorc) [inline profile/README.md]: "build different OSes"

**Review by lorc — APPROVED** [review]
Apart from two minor comments - great job. 

`Acked-by: Volodymyr Babchuk <volodymyr_babchuk@epam.com>`

**Comment by GrygiriiS** [comment]
cool!

File: profile/README.md (hunk 1 above)
Comment (rshym) [reply profile/README.md]: Fixed.

File: profile/README.md (hunk 2 above)
Comment (rshym) [reply profile/README.md]: Fixed.

**Comment by rshym** [comment]
`Ack` is added. Testing is not applicable.
Ready to be merged.

**Pull request merged**